helm/README.md

```diff
@@ -0,0 +1,104 @@
+# ucentralgw
+
+This Helm chart is required for deployment of uCentralGW to the Kubernetes clusters. It is mainly used in [assembly chart](https://github.com/Telecominfraproject/wlan-cloud-ucentral-deploy/tree/main/chart) as uCentralGW requires other services as dependencies that are considered in that Helm chart. This chart is purposed to define deployment logic close to the application code itself and define default values that could be overriden during deployment.
```

Comment:
> This Helm chart is required for deployment of uCentralGW to the Kubernetes clusters.
Wouldn't say it's required but it helps to deploy on Kubernetes.

Reply:
Good point, changed that line